src/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone Project</title>
+    <script
+      src="https://kit.fontawesome.com/deef7307a0.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+    />
+  </head>
+  <body>
+    <!-- Header  -->
+    <header>
+      <nav class="navbar d-flex">
+        <!-- <a href="#"> -->
+        <img
+          class="logo"
+          src="https://raw.githubusercontent.com/SulimanJoya/JavaScript-capstone-project/homepage/src/assets/logo.png"
+          alt="logo"
+        />
+        <!-- </a> -->
+        <ul class="navlinks d-flex">
+          <li class="navitems"><span class="recipe-count"></span></li>
+          <li class="navitems"><a href="#Home">Home</a></li>
+          <li class="navitems"><a href="#Home">Contact Us</a></li>
+          <li class="navitems"><a href="#Home">About</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!------Main------>
+    <section class="homepage"></section>
+    <!------Footer------>
+    <footer class="footer">
+      A Microverse Project, Created by
+      <a href="https://github.com/DannieMo">Daniel Owiri</a> under CC License
+    </footer>
```

Comment:
- [OPTIONAL] I understand that you have implemented responsiveness for standard mobile sizes but you can not take it for granted that all your users will have those devices' sizes thus you should at least have a fallback(reasonable layout for sizes that are not explicitly handled in your code). 

As a starting point, you can simply add a container to your page so that your content doesn't overflow the viewport.

Here is a simple example that you can use in your header, sections and footer. It will definitely solve that issue.

In your CSS:
```css
.container {
  max-width: 1000px;
  margin: 0 auto;
  padding: 0 15%;
}
```

In your HTML:
```html
 <body>
    <!-- Header  -->
    <header>
      <div class="container">
        <nav class="navbar d-flex">
          <!-- <a href="#"> -->
            <img
              class="logo"
              src="https://raw.githubusercontent.com/SulimanJoya/JavaScript-capstone-project/homepage/src/assets/logo.png"
              alt="logo"
            />
            <!-- </a> -->
            <ul class="navlinks d-flex">
              <li class="navitems"><span class="recipe-count"></span></li>
              <li class="navitems"><a href="#Home">Home</a></li>
              <li class="navitems"><a href="#Home">Contact Us</a></li>
              <li class="navitems"><a href="#Home">About</a></li>
            </ul>
        </nav>
      </div>
    </header>
    <!------Main------>
    <section class="homepage">
      <div class="container">
         <!-- Inject your content here -->
      </div>
    </section>
    <!------Footer------>
    <footer class="footer">
      <div class="container">
          A Microverse Project, Created by
          <a href="https://github.com/DannieMo">Daniel Owiri</a> under CC License
     </div>
    </footer>
```